### ISSUE TITLE
How to embed custom data in serenity single page emailable html report

### ISSUE THREAD

**Issue by chnmdey1** [serenity-bdd/serenity-core#2731]
Hi @wakaleo and Serenity team, I want to embed custom data (variable values) in serenity single page emailable report.
In the report I want to print a java variable value from a test scenario as an info, irrespective of pass or failure.

Do we have any way as we have for living documentation, using Serenity.recordReportData()?

**Comment by LucaLis** [comment]
Hi. 

Just a question - is Serenity.recordReportData() docuomented somewhere in Serenity book or guides? It's quite nice feature (i'm going to use it to embedd xhr details fired during step) and it would be good to inform serenity users about this kind of functionality. I can see that custom report fields are documented but it seems that this thing is missing...

**Comment by wakaleo** [comment]
No, but it will be in the new documentation site that will be released soon
